tests/test_ujson.py

```diff
@@ -1,9 +1,11 @@
+import datetime
```

Comment:
Let's follow this pattern: 
https://adamj.eu/tech/2019/09/12/how-i-import-pythons-datetime-module/


```suggestion
import datetime as dt
```